### PR TITLE
Added cbi aggregator build support

### PR DESCRIPTION
**Pull request by sdkrach**
To activate profile create updatesite-aggr folder and put a release.aggr and a nightly.aggr configuration in. Unfortunately, Maven only allows for a single file activation condition, therefore it is not possible to check for the presence of aggregator model and updatesite category.xml.

### CODE REVIEW

File: eclipse/pom.xml [hunk 1]
```diff
@@ -519,6 +521,64 @@
...
+							<dependency>
+							  <artifactId>org.eclipse.equinox.p2.core.feature</artifactId>
+							  <type>eclipse-feature</type>
+							</dependency>
+						  </dependencies>
+						  <repositories>
+							<repository>
+							  <id>cbi-aggregator</id>
+							  <layout>p2</layout>
+							  <url>https://download.eclipse.org/cbi/updates/aggregator/headless/4.13/I20200625-1232/</url>
```
Comment (seiferma) [inline eclipse/pom.xml]: I would favor a non-secure URL, so we can intercept the call with our reliability enhancing proxy.

We should find another solution for this at some point in time...

File: eclipse/pom.xml [hunk 2]
```diff
@@ -519,6 +521,64 @@
...
+							  <type>eclipse-feature</type>
+							</dependency>
+							<dependency>
+							  <artifactId>org.eclipse.equinox.core.feature</artifactId>
+							  <type>eclipse-feature</type>
+							</dependency>
+							<dependency>
+							  <artifactId>org.eclipse.equinox.p2.core.feature</artifactId>
+							  <type>eclipse-feature</type>
+							</dependency>
```
Comment (seiferma) [inline eclipse/pom.xml]: Indentation is mixed between whitespaces and tabs.

**Review by seiferma — CHANGES_REQUESTED** [review]
Looks good to me. I just got two minor comments.

File: eclipse/pom.xml (hunk 1 above)
Comment (sdkrach) [reply eclipse/pom.xml]: Right. I forgot.

File: eclipse/pom.xml (hunk 2 above)
Comment (sdkrach) [reply eclipse/pom.xml]: True

**Review by seiferma — APPROVED** [review]

**Pull request merged**